spec.html

```diff
@@ -11201,6 +11302,9 @@ <h2>Syntax</h2>
         <h1>Static Semantics: MV</h1>
         <p>A numeric literal stands for a value of the Number type or the BigInt type.</p>
         <ul>
+          <li>
+            <emu-grammar>NumericLiteralSeparator :: `_`</emu-grammar> has no MV.
```

Comment:
Should this be an editorial note instead of a normative clause for the MV semantics? Is MV ever "invoked" on NumericLiteralSeparator?